modules/react/skeleton/lib/parts/SkeletonText.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {
+  calc,
+  createStencil,
+  CSProps,
+  cssVar,
+  handleCsProp,
+  px2rem,
+} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
+
+export interface SkeletonTextProps {
+  /**
+   * The number of "lines" that SkeletonText will display. If there is more than one line, the last line will have a width of `60%`.
+   * @default 2
+   */
+  lineCount?: number;
+  /**
+   * The background color of the skeleton
+   * @default `system.color.bg.alt.strong`
+   */
+  backgroundColor?: string;
+}
+
+export interface SkeletonTextLineProps extends CSProps {
+  /**
+   * The width of the line in `px` or `%`.
+   */
+  width?: number | string;
+  /**
+   * The background color of the skeleton
+   * @default `system.color.bg.alt.strong`
+   */
+  backgroundColor?: string;
+  /**
+   * The borderRadius of the shape in `px` or `%`.
+   * @default 2px
+   */
+  borderRadius?: number | string;
+}
```

Comment:
```suggestion

```
i think maybe this was a copy pasta? here's the original props https://github.com/Workday/canvas-kit/blob/master/modules/react/skeleton/lib/parts/skeletonText.tsx